tests/virt_autotest/libvirt_routed_virtual_network.pm

```diff
@@ -62,32 +65,20 @@ sub run_test {
         assert_script_run("virt-clone -o $guest -n $guest.clone -f /var/lib/libvirt/images/$guest.clone");
         assert_script_run("virsh start $guest");
         assert_script_run("virsh start $guest.clone");
+
         #figure out that used with virtio as the network device model during
         #attach-interface via virsh worked for all sles guest
-        assert_script_run("virsh attach-interface $guest network vnet_routed --model virtio --live");
-        assert_script_run("virsh attach-interface $guest.clone network vnet_routed_clone --model virtio --live");
-        #Get the Guest IP Address from ROUTED NETWORK
-        if (get_var("XEN") || check_var("HOST_HYPERVISOR", "xen")) {
-            my $mac_routed = script_output("virsh domiflist $guest | grep vnet_routed | grep -oE \"[[:xdigit:]]{2}(:[[:xdigit:]]{2}){5}\"");
-            script_retry "ip neigh | grep $mac_routed | grep -oE \"([0-9]{1,3}[\.]){3}[0-9]{1,3}\"", delay => 60, retry => 6, timeout => 60;
-            $gi_vnet_routed = script_output("ip neigh | grep $mac_routed | grep -oE \"([0-9]{1,3}[\.]){3}[0-9]{1,3}\"");
-            my $mac_routed_clone = script_output("virsh domiflist $guest.clone | grep vnet_routed_clone | grep -oE \"[[:xdigit:]]{2}(:[[:xdigit:]]{2}){5}\"");
-            script_retry "ip neigh | grep $mac_routed_clone | grep -oE \"([0-9]{1,3}[\.]){3}[0-9]{1,3}\"", delay => 60, retry => 6, timeout => 60;
-            $gi_vnet_routed_clone = script_output("ip neigh | grep $mac_routed_clone | grep -oE \"([0-9]{1,3}[\.]){3}[0-9]{1,3}\"");
-        }
-        else {
-            script_retry "virsh net-dhcp-leases vnet_routed | grep -oE \"([0-9]{1,3}[\.]){3}[0-9]{1,3}\"", delay => 60, retry => 6, timeout => 60;
-            $gi_vnet_routed = script_output("virsh net-dhcp-leases vnet_routed | grep -oE \"([0-9]{1,3}[\.]){3}[0-9]{1,3}\"");
-            script_retry "virsh net-dhcp-leases vnet_routed_clone | grep -oE \"([0-9]{1,3}[\.]){3}[0-9]{1,3}\"", delay => 60, retry => 6, timeout => 60;
-            $gi_vnet_routed_clone = script_output("virsh net-dhcp-leases vnet_routed_clone | grep -oE \"([0-9]{1,3}[\.]){3}[0-9]{1,3}\"");
-        }
-        #Confirm ROUTED NETWORK
-        assert_script_run("ssh root\@$gi_vnet_routed 'traceroute $gi_vnet_routed_clone'", 60);
-        save_screenshot;
-        assert_script_run("ssh root\@$gi_vnet_routed_clone 'traceroute $gi_vnet_routed'", 60);
-        save_screenshot;
-        assert_script_run("virsh detach-interface $guest network --current");
-        assert_script_run("virsh detach-interface $guest.clone network --current");
+        $mac1 = '00:16:3e:32:' . (int(rand(89)) + 10) . ':' . (int(rand(89)) + 10);
+        assert_script_run("virsh attach-interface $guest network vnet_routed --model virtio --mac $mac1 --live");
```

Comment:
The same, please try to use with the "netfront", instead of "virtio" in Xen guests